utils/wasm-gen/src/config/syscalls/param.rs

```diff
@@ -127,6 +128,7 @@ impl SyscallsParamsConfig {
             }
             PtrParamAllowedValues::ReservationId => Ptr::Hash(HashType::ReservationId),
             PtrParamAllowedValues::CodeIdsWithValue { .. } => Ptr::HashWithValue(HashType::CodeId),
+            PtrParamAllowedValues::WaitedMessageId => Ptr::Hash(HashType::MessageId),
```

Comment:
ReplyDeposit also uses the Ptr::Hash(HashType::MessageId) signature. Could this lead to conflict?

Reply:
I guess no. It's not required to work for loose syscall, only for precise

Reply:
Yes, but what if the fuzzer generates config with loose ReplyDeposit syscall?